pcsa_GHN/models.py

```diff
@@ -12,14 +10,22 @@
 
 class Contact(models.Model):
     office_name = models.CharField(max_length=200)
-    contact_number = models.BigIntegerField()
+    contact_number = models.CharField(
+        max_length=20,
+        validators=[
+            RegexValidator(
+                r'^\s*(?:\+?(\d{1,3}))?([-. (]*(\d{3})[-. )]*)?((\d{3})[-. ]*(\d{2,4})(?:[-.x ]*(\d+))?)\s*$',
+                message="Please enter a valid phone number",
```

Comment:
Would be great if we should also, so a sample phone number, along with the message, or something like `Please enter a valid phone number. Format <country code><number>`